web-console/4.0-designs/storage/storage.md

```diff
@@ -29,3 +29,47 @@
 ![form for adding and creating storage](img/create-storage-form.png)
 - Selecting `Create Storage` exposes fields for Storage Class, Name, Access Mode, Size, and the label selector checkbox.
 - Only access modes that are compatible should be available to select in this form.
+
+## Set Default Storage Class
+- Users with appropriate privileges can set a Default Storage Class.
+- THERE ARE TWO WAY USER CAN SET A DEFAULT STORAGE CLASS:
+
+### OPTION-1: Kebab menu option:
+
+1. Login to Openshift > Admin > Storage (menu) > Storage classes (sub-menu)
+2. User can see list of ‘Storage classes” [list view]
+3. There will be a label which indicates which SC is a default one.
+4. User selects the SC that he/she would like to make it as a ‘default class’
+5. Click on the kebab menu > select “Edit Default Storage Class” as soon in the screen below:
+
+![Set a Default Storage Class using kebab menu](img/OCP_set_sefault_class_opt_1.png)
```

Comment:
Does it make sense to have the kebab item "Mark as default storage class" and then they don't have to go into the popup to do this? "Edit default storage class" feels like an action that applies to all SCs so it feels a bit out of place to have it in the kebab like this. 
You could still have the dialog popup that advises them that they are switching from "gp2" to "test" and have them confirm.
WDYT?

Reply:
+1 I like the idea @matthewcarleton has here to make it an even easier interaction for the user.

Reply:
Ya I wonder if we made the "default" badge/label a bit more obvious to help enforce which is the default.

Reply:
I used regular body text with PF gray color.